externs/olx.js

```diff
@@ -2672,6 +2672,42 @@ olx.interaction.SelectOptions.prototype.multi;
 
 
 /**
+ * Options for snap
+ * @typedef {{
+ *     features: (Array.<ol.Feature>|ol.Collection.<ol.Feature>|undefined),
+ *     pixelTolerance: (number|undefined),
+ *     source: (ol.source.Vector|undefined)
+ * }}
+ * @api
+ */
+olx.interaction.SnapOptions;
+
+
+/**
+ * Snap to this features
```

Comment:
"these features".

Maybe also add: "Either this option or `source` need to be provided."